libdnf/base/goal.cpp

```diff
@@ -36,7 +35,7 @@ void add_obseletes(const libdnf::rpm::SolvQuery & base_query, libdnf::rpm::Packa
     libdnf::rpm::SolvQuery obsoletes_query(base_query);
     obsoletes_query.ifilter_obsoletes(libdnf::sack::QueryCmp::EQ, data);
     // TODO(jmracek) Replace obsoletes_query.get_package_set(); by more effective approach
```

Comment:
Please could you remove TODO? It is resolved by your change

Reply:
OK. I will remove.